Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -54,36 +44,38 @@ public override void OnPreSceneQuery()
                 Rays = new RayStep[1];
             }
 
+            closestNormal = Rotation * Vector3.forward;
+
             // Check proximity
-            NearInteractionTouchable newClosestTouchable = null;
+            BaseNearInteractionTouchable newClosestTouchable = null;
             {
-                closestDistance = distFront; // NOTE: Start at distFront for cutoff
-                foreach (var prox in NearInteractionTouchable.Instances)
+                closestDistance = float.PositiveInfinity;
+                foreach (var prox in BaseNearInteractionTouchable.Instances)
                 {
                     if (prox.ColliderEnabled)
                     {
-                       float dist = prox.DistanceToSurface(Position);
-                       if (dist < closestDistance)
-                       {
-   
-                           closestDistance = dist;
-                           newClosestTouchable = prox;
-                       }
+                        Vector3 normal;
+                        float dist = prox.DistanceToSurface(Position, out normal);
```

Comment:
Now I'm realizing a better name for this would be DistanceToTouchable, since not all touchables are surfaces.